src/api/status/public/js/build-log/build-header.js

```diff
@@ -29,6 +30,31 @@ function renderSha(compare, after) {
   buildGitSHA.innerText = after.substring(0, 7);
 }
 
+function renderLoadingAnimation(startedDate, stoppedDate, code) {
+  buildLoadingAnimation.classList.remove('fa-spin');
+
+  if (startedDate != null && stoppedDate == null) {
+    buildLoadingAnimation.innerHTML = 'motion_photos_on';
+    buildLoadingAnimation.classList.add('fa-spin', 'fa-5x', 'text-success', 'fa');
+  } else if (code === 0 && stoppedDate != null) {
+    buildLoadingAnimation.innerHTML = 'check_circle';
+    buildLoadingAnimation.classList.add('text-success');
+  } else if (code !== 0 && stoppedDate != null) {
+    buildLoadingAnimation.innerHTML = 'cancel';
+    buildLoadingAnimation.classList.add('text-danger');
```

Comment:
Question. If the build is in-progress, does it have `code`? If not, we only need `code` for this switch.